quantus/metrics/complexity/complexity.py

```diff
@@ -264,3 +250,29 @@ def evaluate_instance(
 
         a = np.array(np.reshape(a, newshape), dtype=np.float64) / np.sum(np.abs(a))
         return scipy.stats.entropy(pk=a)
+
+    def evaluate_batch(
+        self, *, x_batch: np.ndarray, a_batch: np.ndarray, **_
```

Comment:
what is this for?

Reply:
`*` means every positional argument is ignored. 
`_` typically means we don't care about value (here we don't care about all the keyword arguments). 
I have taken this pattern from here https://github.com/google/CommonLoopUtils/blob/main/clu/metrics.py#L711-L718.